app/presenters/back.rb

```diff
@@ -0,0 +1,25 @@
+class Back
+  def initialize(controller_name:, routes: Rails.application.routes)
+    @controller_name = controller_name
+    @helpers = ActionController::Base.helpers
+    @routes = routes
+  end
+
+  def link
+    @helpers.link_to text, @routes.path_for(controller: @controller_name)
+  end
+
+  private
+
+  def text
+    "#{prefix} #{page_description}"
```

Comment:
You could also generate this text using I18n interpolation, e.g in `en.yml`:

`Back to %{page_description}`

Which is invaluable if we were translating to a language where "Back to" should be last in the sentence.

Reply:
Good shout. I was wondering about that. I guess it makes no difference that the parameter passed to it will be another I18n string.

Reply:
@leeky pushed a commit for this